### PR TITLE
feat(readme): add awesome-typescript

### PR DESCRIPTION
**Pull request by ellerbrock**
https://github.com/ellerbrock/awesome-typescript

Add Awesome-Typescript to this awesome list :)

# By submitting this pull request I confirm I've read and complied with the below requirements.

Failure to properly do so will just result in the pull request being closed and everyone's time wasted. Please read it twice. Most people miss many things.

- [x] I have read and understood the [contribution guidelines](https://github.com/bayandin/awesome-awesomeness/blob/master/CONTRIBUTING.md).
- [x] This pull request has a descriptive title. For example, `Add [Name of List]`, not `Update readme.md` or `Add an awesome list`.
- [x] The list I added **has been around for at least 20 days**,
- [x] The list I added is a non-generated Markdown file in a GitHub repo,
- [x] The list I added is not a duplicate

truly awesome

Cheers Maik

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -72,6 +72,7 @@ A curated list of amazingly awesome awesomeness.
 		- [by @Wolg](https://github.com/Wolg/awesome-swift)
 		- [from ZEEF by @Edubits](https://swift.zeef.com/robin.eggenkamp)
 	- [TypeScript](https://github.com/dzharii/awesome-typescript)
+	  - [by @ellerbrock](https://github.com/ellerbrock/awesome-typescript)
 		- [TypeScript Projects](https://github.com/brookshi/awesome-typescript-projects)
```
Comment (m1guelpf) [inline README.md]: This should be at the same level than your list

**Comment by ellerbrock** [comment]
done

File: README.md [hunk 2]
```diff
@@ -72,7 +72,8 @@ A curated list of amazingly awesome awesomeness.
 		- [by @Wolg](https://github.com/Wolg/awesome-swift)
 		- [from ZEEF by @Edubits](https://swift.zeef.com/robin.eggenkamp)
 	- [TypeScript](https://github.com/dzharii/awesome-typescript)
```
Comment (m1guelpf) [inline README.md]: Also, please remove the link to dzharii's list from the section title, and add it as another subsection, something like.

- Section title (no links)
 - by @oneperson
 - by @anotherperson

**Comment by ellerbrock** [comment]
done

**Pull request merged**